package.json

```diff
@@ -25,12 +25,6 @@
     "check:all": "npm run test && npm run typescript && npm run lint && npm run format:check",
     "prepublishOnly": "npm run check:all && npm run build"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged && npm run typescript && npm run test",
-      "pre-push": "npm run check:all"
-    }
-  },
```

Comment:
Where did these go?

Reply:
I think they were removed in the previous esm-support PR (the one I'm building on top of). Will add it back